ui/perfherder/helpers.js

```diff
@@ -391,7 +391,9 @@ export const getGraphsLink = function getGraphsLink(
 };
 
 // old PhAlerts' inner workings
-const Alert = (alertData, optionCollectionMap) => ({
+// TODO: this shouldn't be exported; configure Jest
+// to forcebly import this before running tests
```

Comment:
What does "forcebly import" mean?

Reply:
I was thinking about testing the Alert function. But now I guess that's not necessary, given it's a private function.